src/main/java/model/RankType.java

```diff
@@ -0,0 +1,79 @@
+package model;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public enum RankType {
+    FIRST(6,2000000000),
+    SECOND(5,30000000),
+    THIRD(5,1500000),
+    FOURTH(4,50000),
+    FIFTH(3,5000);
```

Comment:
```suggestion
    FIRST(6 ,2_000_000_000),
    SECOND(5, 30_000_000),
    THIRD(5, 1_500_000),
    FOURTH(4, 50_000),
    FIFTH(3, 5_000);
```
1. , 뒤에는 공백이 주어져야겠네요.
2. 숫자는 그대로 표기하면 가독성이 떨어지기 때문에 _로 구분지어 숫자 가독성을 올릴 수 있습니다.